pkg/workloads/console/acceptance/acceptance.go

```diff
@@ -186,9 +187,43 @@ func (r *Runner) Run(logger kitlog.Logger, config *rest.Config) {
 			}).Should(Equal(0), "pod did not get deleted")
 		})
 
+		Specify("Fails to run", func() {
```

Comment:
nit: Can we change the name of this? It's a bit ambiguous without reading the test specs to understand what's going on.

Maybe:
> Automatically deleted if not authorised within TTLBeforeRunning

Or something more concise? 😆